src/main/frontend/extensions/code.cljs

```diff
@@ -297,6 +297,8 @@
                 state)
    :did-update (fn [state]
                  (reset! (:code-options state) (last (:rum/args state)))
+                 (.setValue @(:editor-atom state)
+                            (string/join "" (butlast (:lines (last (:rum/args state))))))
```

Comment:
No idea about what `editor-atom` stands for :(
But using `->>` here should make the code easier to read :)